content-services/latest/install/ansible.md

```diff
@@ -0,0 +1,489 @@
+
+Whilst it's possible to override any variable defined by the playbook we have only tested changing the variables defined in `group_vars/all.yml`.
+
+Variables can be overridden using either the `--extra-vars` or `-e` command line option when running the playbook.
+
+If you have more than one variable to override we recommend using a separate file. The file name must be prefixed with "@", for example:
+
+```bash
+ansible-playbook ... --extra-vars "@my-vars.yml"
+```
+
+## AMPs
+
+Several AMP files are downloaded and applied during playbook execution, these are defined in a variable that can be overridden using the mechanism described in the previous section. Follow the steps below to apply your own AMPs
+
+1. Open `group_vars/all.yml` and copy the whole `amp_downloads` variable definition
+2. Create a new file and paste the `amp_downloads` variable
+3. Add any additional AMPs you want to apply paying close attention to the `dest` property. If it's a repository AMP use the `amps_repo` folder, if it's a Share AMP use the `amps_share` folder
+4. Save the file and reference it via the `--extra-vars` option when running the playbook
+
+>NOTE: This mechanism will be improved in a future release.
+
+## JVM Options
+
+Each Java based service installed by the playbook is configured with some default settings, including memory settings.
+
+The defaults are defined in group_vars/all.yml so they can be overridden using the mechanism described [above](#override-playbook-variables).
+
+For example, to override the JAVA_OPTS environment variable for the All In One Transformer Engine place the following in your extra vars file:
+
+```yaml
+tengine_environment:
+  JAVA_OPTS: "$JAVA_OPTS -Xms512m -Xmx1g"
+```
+
+The `*_environment` variable is defined as a dictionary, all keys are added to the relevant components start script thus allowing you to define any number of environment variables.
+
+## External Databases
+
+By default the playbook will install and configure a Postgres server for you. If you'd prefer to use an external database server you can override the `repo_db_url` variable as described [previously](#override-playbook-variables).
+
+An example custom database url is shown below:
+
+```yaml
+repo_db_url: jdbc:mysql://54.164.117.56:3306/alfresco?useUnicode=yes&characterEncoding=UTF-8
+repo_db_driver: com.mysql.jdbc.Driver
+```
+
+Along with the url the database driver binaries need to be provided for one or both services in the `configuration_files/db_connector_repo` and/or `configuration_files/db_connector_sync` folders.
+
+The default database username (`repo_db_username` and/or `sync_db_username`) and password (`repo_db_password` and/or `sync_db_password`) in the configuration file `group_vars/all.yml` can also be overidden with your custom values.
+
+Please refer to the [Configuring Databases](https://docs.alfresco.com/6.2/concepts/intro-db-setup.html) documentation for more detailed information.
```

Comment:
Incorrect link and format: https://docs.alfresco.com/6.2/concepts/intro-db-setup.html